README.md

```diff
@@ -413,22 +417,26 @@ In general, USWDS sets **variables** with **tokens**, and passes those variables
 
 The values of `$theme-font-path` and `$theme-image-path` will be appended to USWDS font paths and image paths, respectively:
 
-```
-$theme-font-path:   '../fonts';
-$theme-image-path:  '../img';
+```scss
+@use "uswds-core" with (
+  $theme-font-path: "../fonts",
+  $theme-image-path: "../img",  
+);
 ```
 
 ## Where things live
-
-- **HTML** markup for the components is located in: `src/components` in the site root. These, however, are written in the templating language Nunjucks. It's best to get HTML source markup directly from [designsystem.digital.gov/components](https://designsystem.digital.gov/components)
-- **Sass** stylesheets are located in: `dist/scss/ (/core, /elements, /components)`. **Compiled CSS** is located in the [downloadable zip file](https://github.com/uswds/uswds/releases/latest).
-- **JS** is located in: `src/js (/components, /utils)`.
+As of USWDS 3.0.0, our codebase is centered around funtional packages, typically components. For more about how we organize packages, [see our Packages documentation](https://designssytem.digital.gov/components/packages). In each of the following examples, we use `[package]` to represent a specific package. For example, component Sass is located in `packages/[package]/src/styles` for an accordion, this would be `packages/usa-accordion/src/styles`.
+- **Template** markup for the components is located in: `packages/[package]/src` in the site root. These, however, are written in the templating language Twig. It's best to get HTML source markup directly from [designsystem.digital.gov/components](https://designsystem.digital.gov/components)
+- **Sass** compoment-specific styles are located in: `packages/[package]/src/styles`. Many components also have a component entry point at `packages/[package]/_index.scss` that includes references to all a component's dependencies as well. **Compiled CSS** is located in the [downloadable zip file](https://github.com/uswds/uswds/releases/latest).
```

Comment:
Clarify location of both source Sass and compiled CSS. I'm not sure why the earlier version linked to the download when referring to the compiled CSS — here I'm linking to the version in `dist`.

Reply:
Same issue with the https://designssytem.digital.gov/components/packages link not resolving in this  bit too @thisisdano

Reply:
Typo in link url. Should be fixed in last push